Gemfile.lock

```diff
@@ -0,0 +1,38 @@
+PATH
```

Comment:
Oops, sorry about that. I've removed it and gitignored it to prevent that from happening in the future.